tools/WindowsFormsMultiWindowSample/src/WindowsFormsMultiWindowSample/Form1.resx

```diff
@@ -0,0 +1,123 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This was saved from VS2022, while our "standard" is still VS2019. Saving from VS2019 changes the file in unexpected ways. Would you please re-save it from VS2019? When we jump to VS2022, we'll revise all of the files in a single PR.